rasa/importers/importer.py

```diff
@@ -7,7 +7,7 @@
 from rasa.core.events import ActionExecuted, UserUttered, Event
 from rasa.core.interpreter import RegexInterpreter, NaturalLanguageInterpreter
 from rasa.core.training.structures import StoryGraph
-from rasa.nlu.constants import MESSAGE_ACTION_NAME, MESSAGE_INTENT_NAME, ACTION_TEXT
+from rasa.nlu.constants import ACTION_NAME, MESSAGE_INTENT_NAME, ACTION_TEXT
```

Comment:
let's also rename intent constant as well
```suggestion
from rasa.nlu.constants import ACTION_NAME, INTENT_NAME, ACTION_TEXT
```